prometheus/doc.go

```diff
@@ -27,6 +27,7 @@
 //
 //    import (
 //    	"net/http"
+//      "log"
```

Comment:
The space just before the double quote needs to be a TAB. And please have this line before the previous one (lexicographical sorting of imports).